st2common/st2common/rbac/types.py

```diff
@@ -184,7 +184,8 @@ def get_resource_type(cls, permission_type):
             return ResourceType.EXECUTION
 
         split = permission_type.split('_')
-        assert len(split) >= 2
+        if not len(split) >= 2:
+            raise ValueError('The length should be greater than or equal to 2.')
```

Comment:
This isn't a good error message, because it hides what the actual issue is, and it doesn't emit what the actual erroneous value is.

The root issue here is that the `permission_type` does not include an underscore. That should be the error message, with the bad value of `permission_type` included in the exception message.